app/helpers/application_helper.rb

```diff
@@ -16,14 +15,21 @@ def resource_class
   def devise_mapping
     @devise_mapping ||= Devise.mappings[:user]
   end
-  def profile(role)
-    if role == "admin"
-      'admin.png'
-    elsif role == "seller"
-      'seller.png'
-    else
-      'buyer.png'
-    end
+
+  def navigation(is_present)
+    return 'layouts/nav/public' if is_present
+
+    'layouts/nav/beforepublic'
```

Comment:
change this name: 'layouts/nav/beforepublic'

Reply:
Changed name